administrator/components/com_admin/tmpl/help/default.php

```diff
@@ -10,44 +10,71 @@
 
 defined('_JEXEC') or die;
 
-use Joomla\CMS\Help\Help;
-use Joomla\CMS\HTML\HTMLHelper;
 use Joomla\CMS\Language\Text;
-use Joomla\CMS\Router\Route;
+use Joomla\Component\Admin\Administrator\View\Help\ToC;
```

Comment:
```suggestion
use Joomla\Component\Admin\Administrator\View\Help\Toc;
```
Fix PHPStan error "Class Joomla\Component\Admin\Administrator\View\Help\Toc referenced with incorrect case: Joomla\Component\Admin\Administrator\View\Help\ToC."